ansible_collections/lvrfrc87/git_acp/tests/tests_own/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Prepare Deploy
+  hosts: localhost
+  connection: local
+
+  collections:
+    - lvrfrc87.git_acp
+
+  tasks:
+  - name: CHECKOUT TO BRANCH-2
+    git_acp:
+      path: "$HOME/Projects/test-automatic-deploy/"
```

Comment:
How does the original module perform tests? This paths are specific to your machine and will probably not work in other environments. If possible, adjust this to match other existing tests. If just relevant locally on your machine (e.g. while developing), eliminate it from the repo.